### PR TITLE
Implement "revalidate" gesture for asynchronous validation

### PR DESCRIPTION
**Pull request by joelverhagen**
Depends on https://github.com/NuGet/NuGetGallery/pull/4677. Address https://github.com/NuGet/Engineering/issues/765.

### CODE REVIEW

File: src/NuGetGallery/UrlExtensions.cs [hunk 1]
```diff
@@ -479,16 +479,27 @@ public static string UploadPackageProgress(this UrlHelper url, bool relativeUrl
                 });
         }
 
-        public static string ReflowPackage(
-            this UrlHelper url,
-            IPackageVersionModel package,
-            bool relativeUrl = true)
```
Comment (loic-sharma) [inline src/NuGetGallery/UrlExtensions.cs]: >relativeUrl [](start = 17, length = 11)

Out of curiosity, why do you remove the `relativeUrl` parameter?

Comment (joelverhagen) [reply src/NuGetGallery/UrlExtensions.cs]: It is unused.

Comment (loic-sharma) [reply src/NuGetGallery/UrlExtensions.cs]: Personally I would rather remain consistent with the other public method signatures than to remove an unused parameter.

Comment (joelverhagen) [reply src/NuGetGallery/UrlExtensions.cs]: Okay, I'll add it back.

Comment (joelverhagen) [reply src/NuGetGallery/UrlExtensions.cs]: Fixed.

**Pull request merged**